proto/context/build.gradle.kts

```diff
@@ -59,10 +59,10 @@ dependencies {
 // Please see `java-tests/runtime` submodule for the tests associated with the Validation Runtime
 // and the generated code.
 //
-modelCompiler {
-    java {
-        codegen {
-            validation().enabled.set(false)
-        }
-    }
-}
+//modelCompiler {
```

Comment:
If the whole thing works together with tests, maybe we could kill these commented portions in this and other files?